<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClerkSetupDialog</class>
 <widget class="QDialog" name="ClerkSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>669</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Clerk And Penner</string>
  </property>
  <widget class="Q3GroupBox" name="groupBox17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>210</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Clerk Order of Inputs</string>
   </property>
   <widget class="Q3Table" name="clerkInputTable">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>22</y>
      <width>190</width>
      <height>270</height>
     </rect>
    </property>
    <property name="numRows" stdset="0">
     <number>10</number>
    </property>
    <property name="numCols" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="clerkBuyerNamesCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Buyer Names</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ticket Length (FF=0)</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="clerkTicketLengthBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="tagSearchBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Tags For Seller</string>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="buttonGroup2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Penner Edit Fields</string>
   </property>
   <widget class="QCheckBox" name="pennerPenEditable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pennerBidEditable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Bid Price</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pennerBuyerEditable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Buyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>620</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ticketHeadingEdit">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>470</y>
     <width>320</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ticketHeadingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ticket Heading:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ticketFormatCheckBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Alternate Ticket Format</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sellerOnTicketBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Seller Name On Ticket</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="printPermBuyersCheckBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Perm Buyers On Ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>620</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="scaleMaxWeightlabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If scale weighs half this amount, or more, Zero Balance will be required.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string extracomment="If scale weighs half this amount, or more, Zero Balance will be required."/>
    </property>
    <property name="text">
     <string>Scale Max Weight: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="scaleMaxWeightTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If scale weighs half this amount, or more, Zero Balance will be required.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="zeroBalNotifycheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Zero Balance Notifications</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabelTopSkip">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip lines from top</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="clerkTicketTopSkipLinesBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="skipSellerIfFoundBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip Seller If Found</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="presortCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Presort ON</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="clerkLoadLastSellerNumberCheckBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Preload Last Seller Number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vetTicketsOnCheckBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vet Tickets ON</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sellingOnLeftcheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>399</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Audience Display
Selling on Left</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
